ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -138,6 +138,10 @@ tests:
   commands: TEST_SUITE=openshift/conformance/parallel run-tests
   openshift_installer:
     cluster_profile: aws
+- as: e2e-aws-fips
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
```

Comment:
I would like to pass the ENABLED_FIPS=true here, but how?? Otherwise, we have to manually edit the job

Reply:
We really need to support limited key value args to these suite definitions, or profiles need to expand much faster.

Reply:
This is going away with the multi-step approach.